kobo/apps/mass_emails/tasks.py

```diff
@@ -228,6 +228,29 @@ def send_emails():
     sender.send_day_emails()
 
 
+@celery_app.task(time_limit=3300)  # 55 minutes
+def _send_emails():
+    """Send the emails for the current day. It schedules the emails if they have not
+    been scheduled yet.
+
+    NOTE: This function will replace the function called send_emails in the near future.
+    """
+    today = timezone.now().date()
+    sender = MassEmailSender()
+    cache_key = f'mass_emails_{today}_emails'
```

Comment:
Can we use a constant for this template so we can keep it consistent between this and `generate_mass_email_user_lists`?